jenkins/run-dbapi.sh

```diff
@@ -79,7 +79,7 @@ conda info -a
 CONDA_ENV_NAME=pyenv-impyla-dbapi-test
 conda create -y -q -n $CONDA_ENV_NAME python=$PYTHON_VERSION pip
 source activate $CONDA_ENV_NAME
-pip install thriftpy sqlalchemy
+pip install thriftpy2 sqlalchemy
```

Comment:
Isn't this going to be an issue with python 2, i.e. we're unconditionally installing thriftpy2 here.

Reply:
thriftpy2 is compatible with python2.x and 3.x, and is considered a replacement for thriftpy, which is deprecated:
https://github.com/Thriftpy/thriftpy2

Reply:
Thanks for the clarification.